client/src/components/Group/GroupPicture.styles.js

```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const baseStyle = {
+    borderWidth: 3,
+    borderRadius: 5,
+    borderColor: '#eeeeee',
+    borderStyle: 'solid',
+    backgroundColor: '#d8d8d8',
+    outline: 'none',
+    transition: 'border .24s ease-in-out'
+};
+  
+export const activeStyle = {
+    borderColor: '#2196f3'
+};
+  
+export const acceptStyle = {
+    borderColor: '#00e676'
+};
+  
+export const rejectStyle = {
+    borderColor: '#ff1744'
+};
```

Comment:
Can we move this whole file so it can be used in one place. I believe these styles are used for the profile img dropzone/dragzone and will be used in the new forum dialogue.